tests/pytest_tests/system_tests/test_reports/test_reports.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+os.environ["WANDB_REPORT_API_ENABLE_V2"] = "True"
+
+import wandb.apis.reports as wr
```

Comment:
This would affect other tests, making tests order-dependent: whether V1 or V2 is picked depends on the environment state at the time of the first import. Python caches imported modules, so after the `reports/__init__.py` runs once, it will not run again.

Instead of this, it's better to just import the version directly:

```suggestion
import wandb.apis.reports.v2 as wr
```